### PR TITLE
Update manage.py

### PR DESCRIPTION
**Pull request by rmb938**
set a custom python version

### CODE REVIEW

**Comment by sjkingo** [comment]
It seems this has broken some tests. Would you be able to update your PR to fix this, and add some tests to properly cover the new changes?

**Comment by rmb938** [comment]
It doesn't seem like those errors are caused by my code. I just tested the master branch and it is getting those same errors.

File: virtualenvapi/manage.py [hunk 1]
```diff
@@ -58,7 +61,10 @@ def _errorfile(self):
 
     def _create(self):
         """Executes `virtualenv` to create a new environment."""
-        proc = subprocess.Popen(['virtualenv', self.name], cwd=self.root, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        if self.python is None:
+            proc = subprocess.Popen(['virtualenv', self.name], cwd=self.root, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        else:
+            proc = subprocess.Popen(['virtualenv', '-p', self.python, self.name], cwd=self.root, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```
Comment (sjkingo) [inline virtualenvapi/manage.py]: Could we move the `['virtualenv', ...]` bit out of the Popen and into the `self.python` conditional so we're not repeating the Popen call twice?

**Comment by sjkingo** [comment]
You're absolutely right - sorry for blaming on you.

**Comment by sjkingo** [comment]
This has been implemented in PR #16 which will be merged later today.

**Pull request closed without merging**